docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -96,6 +96,108 @@ def do_stuff_partitioned():
 
 In addition to the <PyObject object="daily_partitioned_config" decorator /> decorator, Dagster also provides <PyObject object="monthly_partitioned_config" decorator />, <PyObject object="weekly_partitioned_config" decorator />, <PyObject object="hourly_partitioned_config" decorator />.
 
+By default, these decorators create partitions as follows:
+
+<PyObject object="hourly_partitioned_config" decorator /> - a partition is a single
+hour spanning minute 0 to minute 0. For example, defining a partition:
```

Comment:
"defining a partition" makes it sound like a single partition is being defined, but it's actually a PartitionsDefinition, right?

Reply:
oh yeah should probably say that or 'defining a partioned_config'  or something like that